va-gov/components/merger-crosslinks.html

```diff
@@ -36,7 +36,7 @@ <h3 class="va-nav-linkslist-heading">{{ group.heading }}</h3>
           <li>
             <a href="{{link.url}}">
             {% if link.title != empty %}
-              {{ link.title }}
+              {{ link.title }}<img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
```

Comment:
We probably want to add `aria-hidden` to these arrows. They're not useful content for screen readers.